src/Search/Kernel.php

```diff
@@ -178,13 +179,14 @@ public function dependencies(Application $app)
         $app['logger'] = function (Application $app) {
             $logger = new Logger('search-api');
             if ($app['config']['file_log_path']) {
-                $logger->pushHandler(new StreamHandler($app['config']['file_log_path'], Logger::INFO));
+                $stream = new StreamHandler($app['config']['file_log_path'], Logger::INFO);
+                $stream->setFormatter(new LineFormatter());
+                $logger->pushHandler($stream);
             }
             if ($app['config']['file_error_log_path']) {
-                $logger->pushHandler(new StreamHandler($app['config']['file_error_log_path'], Logger::ERROR));
-            }
-            if ($app['config']['loggly_key']) {
-                $logger->pushHandler(new LogglyHandler($app['config']['loggly_key']));
+                $stream = new StreamHandler($app['config']['file_error_log_path'], Logger::ERROR);
+                $stream->setFormatter(new LineFormatter());
```

Comment:
Okay I've switched to the json formatter, I couldn't see that configuration in the journal configuration, but maybe it was just the dev logs I was looking at.